content/01.synopsis.md

```diff
@@ -2,12 +2,12 @@
 
 (Instructions: Describe the background, basic structure of the article, list material to be covered indicating depth of coverage, how they are logically arranged, include recent pubs in the area, 300-500 words)
 
-Substantial technological advances have accelerated basic research and made deep characterization of patient samples routine [TODO: what do we mean by "technological advances" specifically?].
-These technologies provide a rich portrait of genes, cellular pathways, and cell types involved in complex phenotypes.
+The advent of high-throughput profiling methods such as genomics, transcriptomics, and other technologies has accelerated basic research and made deep characterization of patient samples routine.
+These approaches provide a rich portrait of genes, cellular pathways, and cell types involved in complex phenotypes.
 Machine learning is often a perfect fit for extracting disease-relevant patterns from these high dimensional datasets.
 Often, machine learning methods require many samples to identify reproducible and biologically meaningful patterns.
 With rare diseases, biological specimens, and consequently data, are limited due to the rarity of the condition.
 In this perspective, we outline the challenges and emerging solutions for using machine learning in rare disease settings.
-We aim to spur the development of powerful machine learning techniques for rare diseases.
-We also note that precision medicine presents a similar challenge, in which a common disease is partitioned into small subsets of patients with shared etiologies and treatment strategies.
-Advances from rare disease research are likely to be highly informative for other applications as well.
+We also note that precision medicine presents a similar challenge, in which a common disease is partitioned into small subsets of patients with shared etiologies and treatment strategies. [TODO: Are we still talking about precision medicine? I thought we decided against that?]
```

Comment:
To me, this seems worth stating here – even if you work on a common disease that has many subtypes, some of which are rare, this perspective is for you!